strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/main/java/org/carlspring/strongbox/repository/MavenRepositoryFeatures.java

```diff
@@ -57,6 +58,8 @@
 
     private Set<String> defaultArtifactCoordinateValidators;
 
+    private static final String TIMESTAMP_FORMAT = "yyyyMMdd.HHmmss";
```

Comment:
please use `MavenSnapshotManager.TIMESTAMP_FORMAT` instead of new constant.

Reply:
Should I make `TIMESTAMP_FORMAT` as public in `MavenSnapshotManager` to access it in `MavenRepositoryFeatures`?

Reply:
Yeah, I think that will be best!